pkg/crc/machine/vsock.go

```diff
@@ -46,6 +46,7 @@ func isOpened(exposed []types.ExposeRequest, port types.ExposeRequest) bool {
 const (
 	virtualMachineIP = "192.168.127.2"
 	internalSSHPort  = 22
+	hostInterfaceIP  = "127.0.0.1"
```

Comment:
I would drop the `Interface` from the name. Dunno if the right name is `host`, or just `localIP`, `loopbackIP`